app/assets/stylesheets/_global.css.scss

```diff
@@ -42,7 +42,7 @@ label {
 
 // Make all view-all buttons float to the right
 // =============================================================================
-.form-actions .view_all { float: right; }
+// .form-actions .view_all { float: right; }
```

Comment:
So it looks like all of the "View All" links that we do have aren't styled as buttons and don't have the class `.view_all` anyway, so I'm not sure what this was referring to. Can we just take it out?